expression/aggregation/aggregation.go

```diff
@@ -46,15 +41,6 @@ type Aggregation interface {
 
 	// ResetContext resets the content of the evaluate context.
 	ResetContext(sc *stmtctx.StatementContext, evalCtx *AggEvaluateContext)
```

Comment:
can we also remove `GetResult()` function?

Reply:
But `GetPartialResult` method is still used in `mocktikv`. And most of `GetPartialResult` method implementations use `GetResult` method. We cannot remove the unit test because we need it to ensure the correctness of `aggregation` package.
We could replace each `GetResult` calls with `GetPartialResult` in unit tests but it's kind of tedious.

Reply:
Most implementations of `GetPartialResult()` just call the corresponding function `GetResult()`, why not moving the code in `GetResult()` to `GetPartialResult()`?

`GetResult()` is no longer used in the interface, we should remove it.

Reply:
OK, I got you. After remove `GetResult()`, should we replace each `GetResult()` with `GetPartialResult()` in unit tests or just remove the whole unit tests?

Reply:
I prefer this way
> replace each GetResult() with GetPartialResult() in unit tests